packages/meditrak-server/package.json

```diff
@@ -31,6 +31,7 @@
     "http": "^0.0.0",
     "jsonwebtoken": "^7.4.3",
     "lodash": "^4.17.11",
+    "lodash.keyby": "^4.6.0",
```

Comment:
New changes look great! Just wondering do we need this? I was able to use it in meditrak-server without adding the dependency... If it's not required could you remove it?

Reply:
`lodash` contains all of the functions within, so yep, no need - can just `import { keyBy } from 'lodash';` I think?

If you're talking about importing from `lodash.keyby` though @chris-bes, a) there's no need (see above), but b) it would work, because the dependency is in the central `node_modules` thanks to another workspace, but we _still want to be explicit_ about what dependencies each package has (so we don't delete from one and not realise that another package requires it, for e.g.)

Reply:
Fixed. I was having issues with different imports making keyBy sometimes returning a `LodashWrapper` which wasn't what I wanted, didn't realise that importing with curly braces actually makes a difference.